src/warpii.cc

```diff
@@ -142,6 +142,39 @@ void Warpii::setup() {
         enable_floating_point_exceptions();
     }
 
+    prm.declare_entry("Subexpressions", "", Patterns::Map(Patterns::Anything(), Patterns::Anything(), 
```

Comment:
create TODO issue: use `Anything()` pattern instead, and parse this ourselves to avoid awful delimiters.